src/analyzer/analyzerebur128mit.cpp

```diff
@@ -0,0 +1,95 @@
+#include <analyzer/analyzerebur128mit.h>
+#include <QtDebug>
+
+#include "trackinfoobject.h"
+#include "util/math.h"
+#include "util/sample.h"
+#include "util/timer.h"
+
+namespace {
+const double kReplayGain2ReferenceLUFS = -18;
+} // anonymous namespace
+
+AnalyzerEbur128Mit::AnalyzerEbur128Mit(UserSettingsPointer pConfig)
```

Comment:
Is "Mit" referring to the license? I'd prefer that not be in the names of the classes or files...